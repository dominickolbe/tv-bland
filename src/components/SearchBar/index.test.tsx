import { shallow } from "enzyme";
import React from "react";
import { SearchBar } from "./";

describe("<SearchBar />", () => {
  it("renders snapshots", () => {
    const wrapper = shallow(
      <SearchBar
        query=""
        onChangeQuery={(e) => jest.fn()}
        onSubmit={() => jest.fn()}
      />
    );
    expect(wrapper).toMatchSnapshot();
  });
  it("renders snapshots, with query", () => {
    const wrapper = shallow(
      <SearchBar
        query="example search query"
        onChangeQuery={(e) => jest.fn()}
        onSubmit={() => jest.fn()}
      />
    );
    expect(wrapper).toMatchSnapshot();
  });
});
