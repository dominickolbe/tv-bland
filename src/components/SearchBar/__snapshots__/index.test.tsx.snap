// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SearchBar /> renders snapshots 1`] = `
<WithStyles(ForwardRef(Grid))
  container={true}
  spacing={2}
>
  <WithStyles(ForwardRef(Grid))
    item={true}
    xs={12}
  >
    <WithStyles(ForwardRef(Paper))
      className="makeStyles-searchBar-1"
    >
      <WithStyles(ForwardRef(IconButton))
        className="makeStyles-iconButton-3"
        onClick={[Function]}
      >
        <Memo(ForwardRef(SearchIcon)) />
      </WithStyles(ForwardRef(IconButton))>
      <WithStyles(ForwardRef(InputBase))
        autoComplete="on"
        className="makeStyles-input-2"
        onChange={[Function]}
        onKeyPress={[Function]}
        placeholder="Search"
        value=""
      />
      <WithStyles(ForwardRef(Divider))
        className="makeStyles-divider-4"
        orientation="vertical"
      />
      <WithStyles(ForwardRef(IconButton))
        className="makeStyles-iconButton-3"
        onClick={[Function]}
      >
        <Memo(ForwardRef(ClearIcon)) />
      </WithStyles(ForwardRef(IconButton))>
    </WithStyles(ForwardRef(Paper))>
  </WithStyles(ForwardRef(Grid))>
</WithStyles(ForwardRef(Grid))>
`;

exports[`<SearchBar /> renders snapshots, with query 1`] = `
<WithStyles(ForwardRef(Grid))
  container={true}
  spacing={2}
>
  <WithStyles(ForwardRef(Grid))
    item={true}
    xs={12}
  >
    <WithStyles(ForwardRef(Paper))
      className="makeStyles-searchBar-1"
    >
      <WithStyles(ForwardRef(IconButton))
        className="makeStyles-iconButton-3"
        onClick={[Function]}
      >
        <Memo(ForwardRef(SearchIcon)) />
      </WithStyles(ForwardRef(IconButton))>
      <WithStyles(ForwardRef(InputBase))
        autoComplete="on"
        className="makeStyles-input-2"
        onChange={[Function]}
        onKeyPress={[Function]}
        placeholder="Search"
        value="example search query"
      />
      <WithStyles(ForwardRef(Divider))
        className="makeStyles-divider-4"
        orientation="vertical"
      />
      <WithStyles(ForwardRef(IconButton))
        className="makeStyles-iconButton-3"
        onClick={[Function]}
      >
        <Memo(ForwardRef(ClearIcon)) />
      </WithStyles(ForwardRef(IconButton))>
    </WithStyles(ForwardRef(Paper))>
  </WithStyles(ForwardRef(Grid))>
</WithStyles(ForwardRef(Grid))>
`;
